owner/manager.go

```diff
@@ -172,7 +173,21 @@ func (m *ownerManager) CampaignOwner(ctx context.Context) error {
 	return nil
 }
 
+func recoverInOwner(funcName string, quit bool) {
```

Comment:
The function acts like graceful exit process, the function name is not consistent with the behavior.